### DISCUSSION TITLE
bug that updates are not revealed in my academicpage?

### DISCUSSION THREAD

**Discussion by marwa38** [academicpages/academicpages.github.io#2439] [Q&A]
Hi 

I updated my GitHub from here 
![image](https://github.com/user-attachments/assets/e69c0c0d-4c0e-4265-b269-4025aa763dee)

but it is not reflected in my github academicpage but it is still as it was 
![image](https://github.com/user-attachments/assets/21f95855-5e75-4b22-afdc-71a0f9e508ed)

Any advice?

Otherwise please let me know how to change this part in the academicpage?
![image](https://github.com/user-attachments/assets/8638e0bb-ac85-4ed9-b801-89796fd65c7a)

Thanks 
Marwa

**Comment by rjzupkoii** [comment]
In order to update the "About" portion of your Academic Page, you need to edit the `description` section in the `_config.yml`. Updates to your GitHub profile are not propagated since they are managed by GitHub as opposed to in the Academic Page's repository that you forked.

**Comment by marwa38** [comment]
done!
many thanks for this
